pkg/controller/servicebinding/reconciler_test.go

```diff
@@ -189,17 +189,28 @@ func TestReconcilerReconcileUsingSecret(t *testing.T) {
 		require.Equal(t, 1, len(containers))
 		require.Equal(t, 1, len(containers[0].EnvFrom))
 		require.NotNil(t, containers[0].EnvFrom[0].SecretRef)
-		require.Equal(t, reconcilerName, containers[0].EnvFrom[0].SecretRef.Name)
 
 		namespacedName = types.NamespacedName{Namespace: reconcilerNs, Name: reconcilerName}
 		sbrOutput, err := r.getServiceBinding(namespacedName)
 		require.NoError(t, err)
-
 		requireConditionPresentAndTrue(t, CollectionReady, sbrOutput.Status.Conditions)
 		requireConditionPresentAndTrue(t, InjectionReady, sbrOutput.Status.Conditions)
 		requireConditionPresentAndTrue(t, BindingReady, sbrOutput.Status.Conditions)
 
-		require.Equal(t, reconcilerName, sbrOutput.Status.Secret)
+		bindingSecret, err := fakeDynClient.Resource(secretsGVR).Get(sbrOutput.Status.Secret, metav1.GetOptions{})
+		require.NoError(t, err)
+		require.Equal(t, sbrOutput.Status.Secret, bindingSecret.GetName())
+
+		var envList []corev1.EnvFromSource
+		envList = append(envList, corev1.EnvFromSource{
+			ConfigMapRef: nil,
```

Comment:
Pointers are `nil` by default, so it is safe to remove this line as it doesn't add much information.